probes/notCreatedInLast90Days/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: notCreatedInLast90Days
```

Comment:
shall we remove 90 from the probe name and only use it in the `Value` instead? We could call the probe "isRecentRepository", maybe?

Reply:
I am fine with removing the "90" from the name, but removing the "not" brings us the same dilemma as here: https://github.com/ossf/scorecard/pull/3507#discussion_r1381577027

Reply:
Discussed in weekly sync: Remove 90 from probe name.

Reply:
Done in https://github.com/ossf/scorecard/pull/3507/commits/5aed239feecc5bb7173c83ecbe868040d6deb02f.

Reply:
Done in https://github.com/ossf/scorecard/pull/3507/commits/274dff04bc7efd9b7c44f49fb25a7bfe9d92085c